views/js/qtiCreator/editor/MathEditor.js

```diff
@@ -96,36 +101,42 @@ define(['lodash', 'jquery', 'mathJax'], function(_, $, MathJax){
     MathEditor.prototype.renderFromTex = function(){
 
         var args = _processArguments(this, arguments);
+        var _this = this;
```

Comment:
Shall I keep it like this here for consistency with the rest of the file (which is not unit-test covered) ?